.circleci/config.yml

```diff
@@ -2,22 +2,38 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:10
     steps:
       - add_ssh_keys:
           fingerprints:
-            - 'ec:9f:2e:aa:1f:c9:ab:49:57:8f:c0:cd:2e:5b:f3:b0'
+            - "ec:9f:2e:aa:1f:c9:ab:49:57:8f:c0:cd:2e:5b:f3:b0"
```

Comment:
Is your prettier not picking this up?  For me it changes it to single quotes when I save

Reply:
This is probably from the prettier settings being in vscode config. I'll fix.